includes/Runtime.php

```diff
@@ -66,6 +66,12 @@ public function prepareRuntime() {
 	 * Load Runtime into the page.
 	 */
 	public function register_runtime() {
+		\wp_register_script(
+			'nfd-runtime',
+			$this->container->plugin()->url . 'vendor/newfold-labs/wp-module-runtime/includes/runtime.js',
```

Comment:
This should be `null` since we don't need a script file. See https://wordpress.stackexchange.com/questions/298762/wp-add-inline-script-without-dependency or https://github.com/bluehost/bluehost-wordpress-plugin/blob/main/inc/Admin.php#L178-L188

Reply:
We are essentially making the runtime js available for other scripts as a dependency in WP (https://developer.wordpress.org/reference/classes/wp_dependencies/)